### ISSUE TITLE
Add the `no-duplicate-dt` rule

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#1310]
> Within a single dl element, there should not be more than one dt element for each name.

Cite: [4.4.9 The dl element](https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element:~:text=Within%20a%20single%20dl%20element%2C%20there%20should%20not%20be%20more%20than%20one%20dt%20element%20for%20each%20name)

## 👍 Correct

```html
<dl>
  <dt>Apple</dl>
  <dd>lorem...</dd>
  <dd>lorem...</dd>
</dl>
```

## 👎 Incorrect

```html
<dl>
  <dt>Apple</dl>
  <dd>lorem...</dd>
  <dt>Apple</dl>
  <dd>lorem...</dd>
</dl>
```
